### ISSUE TITLE
Can't use 'think' in a loop

### ISSUE THREAD

**Issue by facboy** [shoreditch-ops/artillery#58]
This seems to cause the following:

```javascript
 -/home/me/git/artillery-test/node_modules/artillery/node_modules/artillery-core/lib/engine_util.js:69
    let match = o.match(funcCallRegex);
                 ^

TypeError: Cannot read property 'match' of undefined
    at template (/home/me/git/artillery-test/node_modules/artillery/node_modules/artillery-core/lib/engine_util.js:69:18)
    at f (/home/me/git/artillery-test/node_modules/artillery/node_modules/artillery-core/lib/engine_http.js:37:32)
    at fn (/home/me/git/artillery-test/node_modules/async/lib/async.js:746:34)
    at /home/me/git/artillery-test/node_modules/async/lib/async.js:1213:16
    at /home/me/git/artillery-test/node_modules/async/lib/async.js:166:37
    at /home/me/git/artillery-test/node_modules/async/lib/async.js:706:43
    at /home/me/git/artillery-test/node_modules/async/lib/async.js:167:37
    at /home/me/git/artillery-test/node_modules/async/lib/async.js:1209:30
    at Request.requestCallback [as _callback] (/home/me/git/artillery-test/node_modules/artillery/node_modules/artillery-core/lib/engine_http.js:136:14)
    at Request.self.callback (/home/me/git/artillery-test/node_modules/artillery/node_modules/artillery-core/node_modules/request/request.js:197:22)
```

**Comment by hassy** [comment]
Thanks, this looks like a bug.

**Comment by hassy** [comment]
Fixed in `1.3.5`, thanks for the bug report. :+1:
